ballerina-command/src/main/resources/cli-help/ballerina-help.help

```diff
@@ -1,8 +0,0 @@
-
```

Comment:
@MaryamZi 
We need to keep this file as its referred by the default command. This is used to merge all the help commands together (jballerina + command)

https://github.com/ballerina-platform/ballerina-distribution/blob/release-stage/ballerina-command/src/main/java/org/ballerinalang/command/cmd/DefaultCommand.java#L51

Reply:
Missed that, added it back with 7a88cb7.